decidim-budgets/app/models/decidim/budgets/project.rb

```diff
@@ -36,6 +37,11 @@ def comments_have_votes?
       def confirmed_orders_count
         orders.finished.count
       end
+
+      # Public: Overrides the `notifiable?` Notifiable concern method.
+      def notifiable?(_context)
+        false
```

Comment:
I believe the method refers to the instance: A particular resource might not be notifiable in some circumstances, but the resource itself is.